tox.ini

```diff
@@ -95,3 +95,29 @@ setenv =
 commands =
     pip install -e .
 	behave --tags=-disabled integration_tests
+
+
+[testenv:ci_gsye_122]
```

Comment:
As discussed offline, please put all your code in the `[testenv:ci]` so it gets automatically run in the github actions. Thanks!